electron/legendary/eos_overlay/eos_overlay.ts

```diff
@@ -42,6 +42,10 @@ function getStatus(): {
 
 async function getLatestVersion() {
   if (!existsSync(currentVersionPath)) {
+    // HACK: `overlay_version.json` isn't created when the overlay isn't installed
```

Comment:
As to why this is a `HACK`: In my opinion, requesting the latest overlay version should be possible without it being installed. Currently, Legendary doesn't offer this feature though